lib/web_ui/lib/src/engine/surface/picture.dart

```diff
@@ -19,6 +19,15 @@ const int _kCanvasCacheSize = 30;
 /// Canvases available for reuse, capped at [_kCanvasCacheSize].
 final List<BitmapCanvas> _recycledCanvases = <BitmapCanvas>[];
 
+/// Reduces recycled canvas list by 50% to reduce bitmap canvas memory use.
+void _reduceCanvasMemoryUsage() {
```

Comment:
Comment seems inaccurate. It looks like we're actually deleting all of the recycled canvases.

Maybe this would be a little faster with:

```dart
for (int i = 0; i < canvasCount; i++) {
  _recycledCanvases[i].dispose();
}
_recycledCanvases.clear();
```

Reply:
thx. done.

Reply:
But, is this function used anywhere? :)